components/EditableListItem.tsx

```diff
@@ -9,6 +9,13 @@ export const EditableListItem: React.FC<EditableListItemProps> = (props) => {
   const { t } = useTranslation('common');
   const [isEditing, setIsEditing] = useState(false);
   const [newValue, setNewValue] = useState(props.value);
+  const [overlayOpened, setOverlayOpened] = useState(true);
+
+  useEffect(() => {
+    if (props.credentialRegistryError === 'success') {
```

Comment:
`credentialRegistryError` name seems little off.  Checking error is success or not doesn't seem right

Reply:
**credentialRegistryError** is changed to **credentialRegistryResponse**